src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import cn from 'classnames';
+import { useTodoContext } from '../../context/TodoContext';
+
+export const TodoFooter = React.memo(() => {
```

Comment:
As a rule of thumb, you can extract it from react in imports 

`import React, {memo} from 'react';`